test/BlockCipher/McryptTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace ZendTest\Crypt\BlockCipher;
+
+use Zend\Crypt\Symmetric;
+use Zend\Crypt\BlockCipher;
+
+class MCryptTest extends AbstractBlockCipherTest
+{
+    public function setUp()
+    {
+        try {
+            $this->cipher = new Symmetric\Mcrypt([
+                'algorithm' => 'aes',
+                'mode'      => 'cbc',
+                'padding'   => 'pkcs7'
+            ]);
+        } catch (Symmetric\Exception\RuntimeException $e) {
+            $this->markTestSkipped($e->getMessage());
```

Comment:
@weierophinney basically here we are testing if the Mcrypt extension is enabled or not. We are not testing any functions of Mcrypt here so we cannot expect issue here.